Rawl_B502/dend_rainn_eter_macco_ailella_flainn_bic.trig

```diff
@@ -98,13 +98,16 @@
 <#FiachachOele>
     a foaf:Person ;
     irishRel:nomName "Fiachach Oele" .
-    
+
 <#LóchMár>
     a foaf:Person ;
     irishRel:nomName "Lóch Már" ;
     irishRel:ancestorOfGroup <#Éoganachta>;
-    rel:childOf <#MaFemis>.
+    rel:childOf <#MaFemis>;
+    rdfs:comment "a quo sunt Éoganachta & Fianna Luigne Úi Dedaid Deocluaid".
     #I am not sure how to interpret "Fianna Luigne Úi Dedaid Deocluaid m. Fiachach Oele" - CY
+    #It is ambiguous. Fiacha Oele could be the father of Dedad Deocluaid or the father of Ma Femis.
+    #And the idea of Fianna descending from someone as if they are a kin-group is interesting. EPT
```

Comment:
I wonder if we should interpret this as: "by which are the Éoganachta and the fianna of Luigni of Úi Dedaid Deocluaid".  Basically he is the ancestor of both the Éoganchta and is the ancestor of a Fian band of warriors?

Reply:
Sure, I agree. But the problem is how to divide up the various noun phrases:

1. Lóch Már mc Ma Femis - straightforward.
2. a quo sunt Éoganachta 
3. & Fianna Luigne Úi Dedaid Deocluaid
4. m. Fiachach Oele 

Does 4. follow on immediately from 1. (with 2. and 3. effectively in commas) or from 3. (i.e. Dedad Deocluaid is the son of Fiacha Oele)?

Reply:
Yeah, you are right.  We should probably leave it as it is and come back to it.  Probably worth another blog post!

Reply:
Great idea! :-D